frontend/src/main/java/com/google/tripmeout/frontend/places/PlaceService.java

```diff
@@ -0,0 +1,12 @@
+package com.google.tripmeout.frontend.places;
+
+import java.lang.Exception;
+
+public interface PlaceService {
+  /**
+   * check that a placeId is valid based on the Places API
+   *
+   * @param placeId the id of the place to vaildate
+   */
+  void validatePlaceId(String placesApiPlaceId) throws Exception;
```

Comment:
This should probably throw a more precise validation exception (and possibly IOException, if appropriate).